README.md

```diff
@@ -80,6 +80,12 @@ will get the current forecast, `wed` will get the daily forecast and `weh` will
 
 ![Hourl weather temp only](./assets/hourly_temp_only.png)
 
+## Settings
+The weather-command now has the ability to save settings to default certain flags. The list of possible settings can be seen with:
+
+```sh
+weather-command settings --help
```

Comment:
An update was just released that has changed the command from `weather-command` to `weather`

```suggestion
weather settings --help
```